src/views/AuthorityView/AuthorityView.js

```diff
@@ -59,31 +68,53 @@ const AuthorityView = ({
     });
   };
 
+  const hasEditPermission = () => {
+    const { stripes } = PropTypes;
+
+    return stripes.hasPerm('ui-marc-authorities.authority-record.edit');
+  };
+
+  const toggleAllSections = (expand) => {
+    setCurState(() => {
+      const sections = expandAllFunction(curState, expand);
+
+      return { sections };
+    });
+  };
```

Comment:
Correct me if I'm wrong but I'm pretty sure we don't have Collapse/Expand all button when viewing Authority record. Consequently, we don't need handlers for this.